examples/fully-private-eks-cluster/vpc/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "region" {
+  type        = string
+  description = "AWS region"
+  default     = "us-east-1"
```

Comment:
I can see us-west-2 also used in this example. Can we make the region consistent across the example?